### ISSUE TITLE
Usability: NVDA: Position and Size of list item in 'package Manager' window is not announced by NVDA

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#396]
### Steps to Reproduce

1. Open "Xamarin Workbook"
2. Products window is opened.
3. Navigate to the "wpf "icon and select "create workbook".
4.Under "File" menu select "Add packages" or select 'Add package' button at the bottom of the workbook page.
5. Navigate through list item and observe NVDA announcement

### Expected Behavior
Position and Size of list item should be  properly announced by NVDA
### Actual Behavior

Narrator Behavior: Narrator is announcing as ''Xamarin.Interactive.NuGet.PackageViewModel, 1 of 39 Selected''
NVDA behavior:NVDA is announcing as ''Xamarin.Interactive.NuGet.PackageViewModel, list item focus not Selected'
### User Impact
Users who rely on screen reader will not be able to know the position and size of list items
### Screenshots
![list_item_position_size_inspect](https://user-images.githubusercontent.com/22956130/40776245-d9eafe6a-64e7-11e8-9381-d83cd6684566.png)
![list_item_position_size_narrator](https://user-images.githubusercontent.com/22956130/40776246-da131cf6-64e7-11e8-93dd-dc49552a3a8e.png)
![list_item_position_size_nvda](https://user-images.githubusercontent.com/22956130/40776248-da461bc4-64e7-11e8-8b25-b60d6fb6245b.png)

### Environment
Windows :Widows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)
NVDA Version: 2018.1

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yUsable
Accessibility
Xamarin-Workbooks-Windows
Wipro
